io/net/tcpdump.py

```diff
@@ -66,7 +66,7 @@ def test(self):
             if "packets dropped by interface" in line:
                 if int(line[0]) >= (int(self.drop) * int(self.count) / 100):
                     self.fail("%s, more than %s percent" % (line, self.drop))
-                print line
+                self.log.info(line)
```

Comment:
@harish-24 though this should not be part of this commit, can you change it like:
```
            if "packets dropped by interface" in line:
                self.log.info(line)
                if int(line[0]) >= (int(self.drop) * int(self.count) / 100):
                    self.fail("%s, more than %s percent" % (line, self.drop))
```

Reply:
@narasimhan-v Done.